CHANGELOG.md

```diff
@@ -38,6 +38,10 @@
  * Note that `email_verified == not email_verification_required`.
  * A data migration will be necessary.
 
+## v8.1.1
+
+* Bugfix: Don't show "passwords do not match" when the first password is invalid.
```

Comment:
Looks like the change happened here: https://github.com/incuna/django-user-management/commit/fefdf6a3b9044178ebbe269b3a1f2c1ab1b7b170/